src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -39,7 +39,7 @@ private static void TryLoadManagedAssembly()
                 if (assembly != null)
                 {
                     // call method Datadog.Trace.ClrProfiler.Instrumentation.Initialize()
-                    var type = assembly.GetType("Datadog.Trace.ClrProfiler.Instrumentation", throwOnError: false);
+                    var type = assembly.GetType("OpenTelemetry.Instrumentation.ClrProfiler.Instrumentation", throwOnError: false);
```

Comment:
I think that this change is wrong. The name of the assembly has changed. Not the type's FullName.

Reply:
Yep, the line above should be changed, my bad 🌵

Reply:
Fixed